src/main/java/org/localhost/wmsemployee/controller/SupervisorCommandController.java

```diff
@@ -0,0 +1,47 @@
+package org.localhost.wmsemployee.controller;
+
+import jakarta.validation.Valid;
+import org.localhost.wmsemployee.dto.EmployeeDataDto;
+import org.localhost.wmsemployee.dto.EmployeeRoleUpdateDto;
+import org.localhost.wmsemployee.dto.EmployeeStatusUpdateDto;
+import org.localhost.wmsemployee.dto.UpdateEmployeeDto;
+import org.localhost.wmsemployee.model.Employee;
+import org.localhost.wmsemployee.service.EmployeeCommandService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/supervisor")
+@Validated
+public class SupervisorCommandController {
```

Comment:
podziel to jeszcze na pakiety abyś miał osobne dla pracownika i kierownika i dopiero w tych pakietach rozbijaj na kontroler itp